### ISSUE TITLE
Missing "Post Modifiers" option

### ISSUE THREAD

**Issue by rendetto** [nortikin/sverchok#4175]
Since "Objects In" and "Object ID Out MK2" are merged now, what is the alternative for "Post Modifiers" option?
![ObjectIDOutMK2](https://user-images.githubusercontent.com/66558924/122643915-19aca880-d11b-11eb-9304-0b1f14271767.jpg)

I searched for other methods to integrate a modifier into Sverchok, but it seems that we need to create custom scripts instead:
http://www.codeplastic.com/2019/02/16/sverchok-modifier/

**Comment by vicdoval** [comment]
https://user-images.githubusercontent.com/10011941/121942654-03849e00-cd51-11eb-93c2-3ab4795fd0de.png
The `Post`  button has that funcionality

**Comment by Durman** [comment]
It is not the option which you are looking for?
![2021-06-19_17-52-06](https://user-images.githubusercontent.com/28003269/122644615-27682b00-d127-11eb-8273-6f0090a0b115.png)

**Comment by rendetto** [comment]
@Durman and @vicdoval, Thanks for pointing this out. Closing.

**Comment by rendetto** [comment]
Can this option work while we're in Edit Mode?

**Comment by vicdoval** [comment]
Is it possible to do it via script?

**Comment by rendetto** [comment]
The thing is that I need it to work while Live Update is on. 

What I noticed is that if I enter edit mode, despite "Post" is ON the modifier is not applied. I tried to cheat by making a linked duplicate and enter Edit Mode in that duplicate, while the original stays in Object Mode. The moment I entered Edit Mode at the duplicate, Blender stopped applying the modifier for the original object. So I guess it's not that simple. 

The only solution I got is to make Hooks for the vertices I want to edit. This way I don't need to go Edit Mode, and the modifier is applied.

**Comment by zeffii** [comment]
it may be possible using a proxy object, which is identical to the main object except that it remains in `object mode`, but where all changes are transferred from the proxy's basemesh/curve to the main basemesh/curve upon each update. 
```
if in_edit_mode(obj) and self.apply_modifiers:
     # throttle sverchok
     #   make or pickup (temporary) proxy, doing full copy of obj and modifier stack
     #   get post mesh from proxy
     #   append mesh to outputs
     #   hide/remove proxy

```
maybe.

**Comment by zeffii** [comment]
i think there's a good chance that it's visually disturbing to see the post modifier result at the same time as editing the object,

**Comment by rendetto** [comment]
> there's a good chance that it's visually disturbing to see the post modifier result at the same time as editing the object

It depends on the setup I guess. Blender itself offers that option:
![DisplayModifierInEditMode](https://user-images.githubusercontent.com/66558924/122654899-12a38b80-d157-11eb-8448-dc880015f1a1.jpg)
I'm used to it and find it useful in some cases. But in case it's too complicated to be implemented, it's perfectly fine for me to stick to the hooks method I used before.

**Comment by zeffii** [comment]
> Blender itself offers that option:

yeah, it would no doubt have valid use-cases.  There's only one way to find out out if this is a useful feature : )
